### DISCUSSION TITLE
How to update page tags using graphql

### DISCUSSION THREAD

**Discussion by benjamin-bergia** [requarks/wiki#6704] [Help / Questions] [answered]
I have been trying to update the list of tags from a page via the API and it looks like I can't update just the tags:
```graphql
mutation {
  pages {
    update (id:3, tags:["test"]){
      responseResult {
        message
      }
      page {
        tags {
          id
          tag
          title
        }
      }
    }
  }
}
```
```json
{
  "data": {
    "pages": {
      "update": {
        "responseResult": {
          "message": "Page content cannot be empty."
        },
        "page": null
      }
    }
  }
}
```

 Is this the expected behavior?
 If I update the content it does work but that doesn't make much sense to do so.

**Comment by NGPixel** [comment, marked as answer]
The current implementation requires all fields to be present when updating a page. This has been addressed in v3.
